src/style.css

```diff
@@ -1 +1,53 @@
-/* Styles go here */
+.header {
+  display: flex;
+  padding: 0 50px;
+  font-family: Roboto, sans-serif;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  display: inline-flex;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  height: 15px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 22.5px 0;
+  color: black;
+  position: relative;
+}
```

Comment:
try this to center the text inside link

```suggestion
.nav__link {
  height: 60px;
  line-height: 60px;
  font-size: 12px;
  text-transform: uppercase;
  text-decoration: none;
  color: black;
  position: relative;
}
```

Reply:
Thanks it's works
Tests had passed by npm but not passed on github (

![изображение](https://user-images.githubusercontent.com/92954488/161954606-9b036786-7507-4392-ad40-28826c04e330.png)